save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ProjectController.kt

```diff
@@ -72,19 +74,53 @@ class ProjectController(private val projectService: ProjectService,
      * FixMe: requires 'write' permission, because now we rely on this endpoint to load `ProjectView`
      *
      * @param name name of project
-     * @param owner owner of project
      * @param authentication
-     * @return project by name and owner
+     * @param organizationId
+     * @return project by name and organization
      * @throws ResponseStatusException
      */
     @GetMapping("/get")
     @PreAuthorize("hasRole('VIEWER')")
     @Suppress("UnsafeCallOnNullableType")
-    fun getProjectByNameAndOwner(@RequestParam name: String,
-                                 @RequestParam owner: String,
-                                 authentication: Authentication,
+    fun getProjectByNameAndOrganization(@RequestParam name: String,
```

Comment:
```suggestion
    fun getProjectByNameAndOrganizationId(@RequestParam name: String,
```

Reply:
Also may be rename end point from simple `/get` to something more meanable

Reply:
Fixed